index.html

```diff
@@ -11,5 +11,6 @@
   <body>
     <div id="root"></div>
     <script type="module" src="/src/index.jsx"></script>
+    <script src="/code/kiri_pool.js"></script>
```

Comment:
Can we put a comment here explaining what this is, where it comes from, and who deserves credit for it?

Reply:
ok not using this script :) so I put the credit to the API in the kirimoto.jsx file